src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -713,20 +712,25 @@ def show_blob(cmd, client, container_name, blob_name, snapshot=None, lease_id=No
 def storage_blob_delete_batch(client, source, source_container_name, pattern=None, lease_id=None,
                               delete_snapshots=None, if_modified_since=None, if_unmodified_since=None, if_match=None,
                               if_none_match=None, timeout=None, dryrun=False):
+    container_client = client.get_container_client(source_container_name)
+
     @check_precondition_success
     def _delete_blob(blob_name):
         delete_blob_args = {
-            'container_name': source_container_name,
-            'blob_name': blob_name,
-            'lease_id': lease_id,
+            'blob': blob_name,
+            'lease': lease_id,
             'delete_snapshots': delete_snapshots,
             'if_modified_since': if_modified_since,
             'if_unmodified_since': if_unmodified_since,
             'if_match': if_match,
             'if_none_match': if_none_match,
             'timeout': timeout
         }
-        return client.delete_blob(**delete_blob_args)
+        try:
+            container_client.delete_blob(**delete_blob_args)
+            return blob_name
+        except HttpResponseError:
+            pass
```

Comment:
```suggestion
        except HttpResponseError:
            logger.debug(f"delete blob failure: {error}")
            return None
```
In case customers want to know why the deletion fails and we can guide them to see debug info